1.1.0/jessie/build-projectjson/README.md

```diff
@@ -11,8 +11,10 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 
 - [`1.1.0-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-projectjson/Dockerfile)
```

Comment:
I'm not sure how deliberate the sort order is. @glennc is the original author and he chose descending. I like it too as it makes the 'latest' tag more prominent.